### PR TITLE
Emphasize the need to specify port for network access

### PR DESCRIPTION
**Pull request by radinamatic**
As reported by @laurenlichtman, it was not clear enough that the server port needed to be specified when accessing from client devices. Screenshot was also updated.

Before | After
------- | ------
device-info-old-300 | device-info-300

### CODE REVIEW

**Review by indirectlylit — APPROVED** [review]

**Comment by radinamatic** [comment]
Thanks! 🙏

File: docs/access.rst [hunk 1]
```diff
@@ -89,9 +89,9 @@ Kolibri currently supports the following combinations of operating systems and b
 Set up Access on Other Devices
 ******************************
 
-To access content from other devices in the same network, you need to know the :ref:`IP address <ips>` of the computer where Kolibri is running. 
+To access content from other devices in the same network, you need to know the :ref:`IP address <ips>` of the computer where Kolibri is running. The default port is **8080**, and you must add it after the IP address.
 
-For example, if Kolibri is installed and started on a computer with the IP address **192.168.0.104**, you can access it from an Android tablet connected to the same network by opening the browser on the tablet and typing the address ``http://192.168.0.104:8080``.
+For example, if Kolibri is installed and started on a computer with the IP address **192.168.8.134:8080**, you can access it from an Android tablet connected to the same network by opening the browser on the tablet and typing the address ``http://192.168.8.134:8080``.
```
Comment (benjaoming) [inline docs/access.rst:94]: > a computer with the IP address **192.168.8.134:8080**

This seems wrong, as it isn't an IP address with the `:8080` notation added.

Comment (radinamatic) [reply docs/access.rst:94]: Not sure I understand what you mean...?

Comment (benjaoming) [reply docs/access.rst:94]: @radinamatic ^

Comment (benjaoming) [reply docs/access.rst:94]: It should say:

> a computer with the IP address **192.168.8.134**

(deleted `:8080`)

Comment (radinamatic) [reply docs/access.rst:94]: Ok, I see what you mean now, but the idea was to emphasize the need for the port 😉

I'll rephrase to include a separate clause for the port, thanks!

Comment (indirectlylit) [reply docs/access.rst:94]: all that needs to change is "with the IP address" -> "with the address"

Comment (benjaoming) [reply docs/access.rst:94]: That'll work, too

Comment (radinamatic) [reply docs/access.rst:94]: Ok, that was easy https://github.com/learningequality/kolibri-docs/pull/125
Thanks to both! 🙏

**Pull request merged**